cloud: update kafka changefeed doc based on docs usability testing (#10272)
pingcap/docs: tidb-cloud/changefeed-sink-to-apache-kafka.md

3. If you select **Avro** as your data format, you will see some Avro-specific configurations on the page. You can fill in these configurations as follows:

    - In the **Decimal** and **Unsigned BigInt** configurations, specify how TiDB Cloud handles the decimal and unsigned bigint data types in Kafka messages.
    - In the **Schema Registry** area, fill in your schema registry endpoint. If you enable **HTTP Authentication**, the fields for user name and password are displayed and automatically filled in with your TiDB cluster endpoint and password.

4. In the **Topic Distribution** area, select a distribution mode, and then fill in the topic name configurations according to the mode.

>     If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribute Mode** drop-down list.

Author: ```suggestion
    If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribution Mode** drop-down list.
```

>     The distribution mode controls how the changefeed creates Kafka topics, by tables, by databases, or creating one topic for all changelogs.

Reviewer: ```suggestion
    The distribution mode controls how the changefeed creates Kafka topics, by table, by database, or creating one topic for all changelogs.
```

   - **Distribute changelogs by table to Kafka Topics**

>         If you want the changefeed to create a dedicated Kafka topic for each table, choose this mode. Then, all Kafka messages of a table are sent to a dedicated Kafka topic. You can customize topic names for tables by setting a topic prefix, a separator between a database name and table name, and a suffix. For example, if you set the separator as `-`, the topic names are in the format of `<Prefix><DatabaseName>-<TableName><Suffix>`.

Reviewer 2: ```suggestion
        If you want the changefeed to create a dedicated Kafka topic for each table, choose this mode. Then, all Kafka messages of a table are sent to a dedicated Kafka topic. You can customize topic names for tables by setting a topic prefix, a separator between a database name and table name, and a suffix. For example, if you set the separator as `_`, the topic names are in the format of `<Prefix><DatabaseName>_<TableName><Suffix>`.
```
